x/earn/keeper/proposal_handler.go

```diff
@@ -0,0 +1,13 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/kava-labs/kava/x/earn/types"
+)
+
+// HandleCommunityPoolDepositProposal is a handler for executing a passed community pool deposit proposal
+func HandleCommunityPoolDepositProposal(ctx sdk.Context, k Keeper, p *types.CommunityPoolDepositProposal) error {
+	// deposit from community pool address (kava1jv65s3grqf6v6jl3dp4t6c9t9rk99cd8m2splc)
+	return k.Deposit(ctx, k.accountKeeper.GetModuleAddress("distribution"), p.Amount)
```

Comment:
I beleive we'll need to update the community pool object stored in `distribution` like in `DistributeFromFeePool`: https://github.com/cosmos/cosmos-sdk/blob/v0.45.6/x/distribution/keeper/fee_pool.go.

The distribution module account holds the community pool plus pending staking rewards, so if we don't update it we could deposit too much and prevent user's from being able to claim.